drgn/commands/_builtin/crash/ps.py

```diff
@@ -0,0 +1,294 @@
+            CellFormat("ST", ">"),
+            CellFormat("%MEM", ">"),
+            CellFormat("%VSZ", ">"),
+            CellFormat("RSS", ">"),
+            CellFormat("COMM", "<"),
+        )
+    ]
+    tasks.sort(key=lambda t: t.pid.value_())
+    page_size = int(prog["PAGE_SIZE"])
+    total_mem = totalram_pages(prog)
+    for t in tasks:
+        task_rss = get_task_rss_info(prog, t)
+        rss_kb = task_rss.total * page_size // 1024
+        pct_mem: float = task_rss.total / total_mem
+        rows.append(
+            [
+                t.pid.value_(),
+                t.parent.pid.value_(),
+                CellFormat(task_cpu(t), "^"),
+                CellFormat(t.value_(), "^x"),
+                task_state_to_char(t),
+                CellFormat(pct_mem, ".1%"),
+                task_vsize(t) // 1024,
+                rss_kb,
+                (
+                    f"[{escape_ascii_string(t.comm.string_())}]"
+                    if task_is_kthread(t)
+                    else f"{escape_ascii_string(t.comm.string_())}"
+                ),
+            ]
+        )
+    print_table(rows)
+
+
+def check_arg_type(arg: Optional[str]) -> Tuple[str, Any]:
+    """
+    Check the filter type of the argument
+    """
+    if arg is not None:
+        try:
+            return ("pid", int(arg, 10))
+        except ValueError:
+            pass
+        try:
+            return ("task", int(arg, 16))
+        except ValueError:
+            return ("comm", arg)
+    else:
+        return ("none", None)
+
+
+@crash_command(
+    description="process information",
+    long_description="display process status information",
+    arguments=(
+        mutually_exclusive_group(
+            argument(
+                "-u",
+                dest="user",
+                action="store_true",
+                default=False,
+                help="display only user threads information",
+            ),
+            argument(
+                "-k",
+                dest="kernel",
+                action="store_true",
+                default=False,
+                help="display only kernel threads information",
+            ),
+        ),
+        mutually_exclusive_group(
+            argument(
+                "-m",
+                dest="last_run",
+                action="store_true",
+                default=False,
+                help="show last run information",
+            ),
+            argument(
+                "-l",
+                dest="last_run_timestamp",
+                action="store_true",
+                default=False,
+                help="show last run information (raw timestamp)",
+            ),
+        ),
+        argument(
+            "-G",
+            dest="group_leader",
+            action="store_true",
+            default=False,
+            help="display only the thread group leader in a thread group",
+        ),
+        argument(
+            "arg",
+            nargs="?",
```

Comment:
Crash seems to support passing multiple of these and includes the task if it matches any of them.